src/edgeDebugAdapter.ts

```diff
@@ -60,7 +61,45 @@ export class EdgeDebugAdapter extends CoreDebugAdapter {
         return capabilities;
     }
 
-    public launch(args: ILaunchRequestArgs): Promise<void> {
+    private async _checkPortOccupiedByEDP(address: string, port: number): Promise<void> {
```

Comment:
Would it make sense to refactor the code in _getVersionData in vscode-chrome-debug-core to find a way to use that code, instead of creating our own version of calling /json/version here?

Reply:
I think it's not needed since the only thing common is the GET call to /json/version. We want to add add logs/telemetry/errors which are relevant to Edge here hence I think we should keep this here itself, otherwise we might have to do platform checks in core to do so.